### PR TITLE
[TR] PIM-4760 : error if quick export not configured

### PR DESCRIPTION
**Pull request by jmleroux**
| Q                  | A
| ------------------ | ---
| Migration script?  |n
| Behats             |y
| Specs & Static     |y
| Changelog updated  |n

### CODE REVIEW

**Comment by solivier** [comment]
GTM

File: src/Pim/Bundle/DataGridBundle/Resources/public/js/datagrid/widget/export-widget.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 define(
-    ['backbone', 'oro/messenger'],
-    function (Backbone, messenger) {
+    ['jquery', 'underscore', 'backbone', 'oro/messenger'],
+    function ($, _, Backbone, messenger) {
         'use strict';
 
-        var ExportWidget = Backbone.View.extend({
+        return Backbone.View.extend({
```
Comment (phaseinducer) [inline src/Pim/Bundle/DataGridBundle/Resources/public/js/datagrid/widget/export-widget.js]: Is there a particular reason to change that ?

File: src/Pim/Bundle/DataGridBundle/Resources/public/js/datagrid/widget/export-widget.js [hunk 2]
```diff
@@ -12,11 +12,20 @@ define(
             },
 
             run: function () {
-                $.ajax(this.action.getLinkWithParameters());
-                messenger.notificationFlashMessage('success', _.__('pim.grid.mass_action.quick_export.launched'));
+                $.ajax(this.action.getLinkWithParameters())
```
Comment (phaseinducer) [inline src/Pim/Bundle/DataGridBundle/Resources/public/js/datagrid/widget/export-widget.js]: Prefer `$.get()`

File: src/Pim/Bundle/DataGridBundle/Resources/public/js/datagrid/widget/export-widget.js (hunk 1 above)
Comment (jmleroux) [inline src/Pim/Bundle/DataGridBundle/Resources/public/js/datagrid/widget/export-widget.js]: not necessary to instantiate a variable when you directly return it after instantiation.

Comment (phaseinducer) [inline src/Pim/Bundle/DataGridBundle/Resources/public/js/datagrid/widget/export-widget.js]: Oh yes you're right ! I dunno why, I thought you did the reverse thing. My bad.

**Comment by phaseinducer** [comment]
GTM

**Pull request merged**